### PR TITLE
Background for Login/Registration

### PR DESCRIPTION
**Pull request by jfan0010**
# Description 
Added background for login and registration page

# Type of change 
- [ ] Bug fix
- [x] New feature
- [ ] Refactoring
- [ ] Documentation 

# Problem
No bg before

# Solution description
As above

# Tests 
All unit tests pass? Y

Builds successfully? Y

Attach screenshot of the expected result (e.g. UI of the website/ postman api call result/ database changes)

image
image

# Risk
Identity the risk level (low, medium, high) and indicate which component(s) will be affected if bugs exist.
Medium - Low. Might affect registration/login feature.

### CODE REVIEW

File: client/src/components/auth/authComponents/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/**
```
Comment (jfan0010) [inline client/src/components/auth/authComponents/LoginForm.jsx:1]: This file is the same as previous `Login.js`, just that removed the `Don't have account yet? Sign up today!` link beside the button.

**Comment by vgoh1** [comment]
Wow I can't believe how much better register/login looks when we change the background! I just had a few thoughts aesthetically:

What do you think of making the boxes match the ones on the landing page like below?
image

And the login/register buttons as well to match:
image
Or to match the login/register in the top right. Cause the bootstrap blue is looking a bit out of place right now.

**Review by melvinalex7 — APPROVED** [review]
Wow. The Login Page and Register Page look really good now. The transition to the login and register page is much better now—just a few suggestions.

1. 
image
Should we add the nav links on the home page to the login and register page as well? Once you go to log in/register, all other links can only be accessed after going back to the home page.

2. 
image
Also, the register overlay is currently above the footer, which doesn't look right.

**Comment by jfan0010** [comment]
> Wow. The Login Page and Register Page look really good now. The transition to the login and register page is much better now—just a few suggestions.
> 
> 
> image
> Should we add the nav links on the home page to the login and register page as well? Once you go to log in/register, all other links can only be accessed after going back to the home page.
> 
> 
> image
> Also, the register overlay is currently above the footer, which doesn't look right.

1. Yep we can, but the reason is the `nav links` in `landing page header` doesn't really work right now. And if it works, it will actually bring you to a different section on the `landing page` (not to different pages), not quite sure if we still need them if they're all navigating to the `landing page`. But yes I can add them back once the nav links are working.
2. Ops, can you also provide your screen dimension so that I can recreate the same issue and fix it?

**Comment by jfan0010** [comment]
> Wow I can't believe how much better register/login looks when we change the background! I just had a few thoughts aesthetically:
> 
> What do you think of making the boxes match the ones on the landing page like below?
> image
> 
> And the login/register buttons as well to match:
> image
> Or to match the login/register in the top right. Cause the bootstrap blue is looking a bit out of place right now.

I'll change them, btw do you think we should use the "brown gold" colour for the fonts or black as it is now is alright?

**Comment by vgoh1** [comment]
@jfan0010 Black because brown gold looks a bit out of place for normal body text imo.

**Comment by rpra0006** [comment]
> > Wow. The Login Page and Register Page look really good now. The transition to the login and register page is much better now—just a few suggestions.
> > image
> > Should we add the nav links on the home page to the login and register page as well? Once you go to log in/register, all other links can only be accessed after going back to the home page.
> > image
> > Also, the register overlay is currently above the footer, which doesn't look right.
> 
> 1. Yep we can, but the reason is the `nav links` in `landing page header` doesn't really work right now. And if it works, it will actually bring you to a different section on the `landing page` (not to different pages), not quite sure if we still need them if they're all navigating to the `landing page`. But yes I can add them back once the nav links are working.
> 2. Ops, can you also provide your screen dimension so that I can recreate the same issue and fix it?

I think the "Researchify" button actually brings you to the landing page url. I tested it out in Mable's #222 and refreshes the page when you click on it.

**Review by rpra0006 — CHANGES_REQUESTED** [review]
image
I'm using 100% page size and the register form is cutting the footer border and covering the copyright text.

**Review by rpra0006 — CHANGES_REQUESTED** [review]
image
Login page is also a bit large, maybe you can remove some of the border spacing at the top and bottom of login form? The spacing between the landing header to the login form can also be reduced as well.

**Comment by jfan0010** [comment]
Changed the colour of forms and buttons. Should have fixed the footer issue, let me know if it doesn't fix 😢

**Comment by melvinalex7** [comment]
> Changed the colour of forms and buttons. Should have fixed the footer issue, let me know if it doesn't fix 😢

Looks like it is fixed when the zoom is 100% but when it goes to 110% the issue still persists.

**Comment by jfan0010** [comment]
> 
> Looks like it is fixed when the zoom is 100% but when it goes to 110% the issue still persists.

That is weird cause 110% on registration page and 125% on login page still work fine for me. But I think I would fix zooming issue later in the next ticket cause I might need some time to adjust the CSS configuration.

**Review by rpra0006 — CHANGES_REQUESTED** [review]
image
I still have the same issue with registration but login looks great (only happens in laptop resolution but not desktop). If it's hard to fix, we can ignore this part for later?

 Also can you change the color of text inside "login" button, maybe to a white? It's hard to see brown on gold.

**Comment by jfan0010** [comment]
> I still have the same issue with registration but login looks great (only happens in laptop resolution but not desktop). If it's hard to fix, we can ignore this part for later?
> 
> Also can you change the color of text inside "login" button, maybe to a white? It's hard to see brown on gold.

😢 It shouldn't be hard to fix, just that I couldn't recreate the issue 🤣 . But I'll try it again later.

For the `login` button, black font is for the consistency with the `discover more` button on landing page, Not sure if we still wanna change them to white?

**Comment by rpra0006** [comment]
> > I still have the same issue with registration but login looks great (only happens in laptop resolution but not desktop). If it's hard to fix, we can ignore this part for later?
> > Also can you change the color of text inside "login" button, maybe to a white? It's hard to see brown on gold.
> 
> 😢 It shouldn't be hard to fix, just that I couldn't recreate the issue 🤣 . But I'll try it again later.
> 
> For the `login` button, black font is for the consistency with the `discover more` button on landing page, Not sure if we still wanna change them to white?

Idk I just think it's hard to see black on the dark gold color. Maybe someone else has an input?

**Comment by ssha0054** [comment]
> > > I still have the same issue with registration but login looks great (only happens in laptop resolution but not desktop). If it's hard to fix, we can ignore this part for later?
> > > Also can you change the color of text inside "login" button, maybe to a white? It's hard to see brown on gold.
> > 
> > 
> > 😢 It shouldn't be hard to fix, just that I couldn't recreate the issue 🤣 . But I'll try it again later.
> > For the `login` button, black font is for the consistency with the `discover more` button on landing page, Not sure if we still wanna change them to white?
> 
> Idk I just think it's hard to see black on the dark gold color. Maybe someone else has an input?

Yeah I agree with this. Personally I'm not a huge fan of the gold button, I think if it was similar to the Register button in being blue and white it will look better and is more readable. Since the register and log in page doesn't have any text matching the gold button like the landing page does, I reckon it looks quite inconsistent in the register and log in page.

**Comment by jfan0010** [comment]
I actually agree with you, but I was actually trying to match the original style on the `landing page` (top right on `landing page`: blue for `register` and `gold` for login), but because white background with gold font doesn't look good as a button in the form, I changed all the `log in` button to the same as the `discover more` button on the `landing page`. What if I change the login button to the gold background and white fonf, would that works?

image

**Comment by cyr17** [comment]
> I actually agree with you, but I was actually trying to match the original style on the `landing page` (top right on `landing page`: blue for `register` and `gold` for login), but because white background with gold font doesn't look good as a button in the form, I changed all the `log in` button to the same as the `discover more` button on the `landing page`. What if I change the login button to the gold background and white fonf, would that works?
> 
> image

hmm maybe have white text in the gold button in both the discover more and login buttons , and this would make it look like it uses an inverted pallet of the login button from the landing page

**Review by cyr17 — APPROVED** [review]
Looks dope btw!

**Review by rpra0006 — APPROVED** [review]
Don't want to block this PR too long, but functionality wise good

**Comment by jfan0010** [comment]
Have changed the gold buttons to white font, applied some CSS to fix the overlapping issue (should have work now I guess, if not will pass it to others who can recreate the issue.) Will merge in now and maybe improve the CSS in the coming ticket if have time.

**Pull request merged**